### ISSUE TITLE
Editting Project info/metadata (possible bug)

### ISSUE THREAD

**Issue by cwatry** [CTUIR/CDMS#22]
As sa, I am trying to update Project information using the Edit Project option; however, my changes are not being saved.  Is/should this feature (be) limited specifically to owners/editors or should it automatically be granted to sa's as well?  Doesn't seem like an sa should need to be assigned as an editor to be able to edit - possible bug??

**Comment by kenburcham** [comment]
When you say "sa" you're meaning the "admin" user? Can you open the dev tools (F12) and and post any error message/screen shot of what the error is? The original design was that if the owner/editor can do it, the admin can, too.  Did you notice if updates works for the project owner/editor?

**Comment by cwatry** [comment]
Yes, admin.  When I go to edit (e.g., I added the text “xyz” to Project Statement/Goal)  and save, I do not get an error message, I simply see that the change I made is not displayed.  However, I was just able to make a change and it did save, but when I went back and tried to delete the text I added, it won’t save the change again.  I’ve tried refreshing my browser several times now and the text is still visible and unchanged.

Here’s a shot of the dev-tools and then the error when I select the specific error message…

[cid:image001.png@01D449C8.2079B140]


[cid:image002.png@01D449C8.597F9F90]

**Comment by kenburcham** [comment]
Hey Clark,

Can you email me that picture directly? I’m not getting it. (or post in comments in github)

**Comment by kenburcham** [comment]
Looks like this does save, but only intermittently. So this is an ugly bug.
